SDK and tests for hybrid RP

farehar opened the pull request:
https://github.com/Azure/azure-rest-api-specs/pull/7568

Commit:
hybrid RP sdk

isra-fel commented on sdk/hybridcompute/Microsoft.Azure.Management.HybridCompute/src/Microsoft.Azure.Management.HybridCompute.csproj:
```diff
@@ -0,0 +1,25 @@
+    <PackageTags>management;hybridcompute;</PackageTags>
+    <PackageReleaseNotes>
+      <![CDATA[
+        This is private preview release of the Azure Hybrid Compute SDK.
```
Hey @farehar , are you sure if this is for private preview? It should be against azure/sdk-for-net-pr repo if so.

farehar replied:
@isra-fel: You are right, this is for our public preview.

Commit:
Add delete operation

Commit:
update release information

isra-fel reviewed (changes requested):
Hi @farehar ,

- could you put a link to the swagger review PR(s) in description
- As a public preview SDK, its spec should also be in the public spec repo. Please update `generate.ps1` so that it doesn't point to the private spec repo
- And you'll see a .txt file after running `generate.ps1`, please also check it in
- please also check in `AssemblyInfo.cs`, and make sure the versions in it are updated

Thanks

isra-fel commented:
@farehar ping on this

farehar commented:
@isra-fel : One of the feedback was to move our swagger spec to public repository. We weren't ready to do that last month. I've created a pull request in public repository today. If i address rest of the feedback, would we be able to merge before swagger spec PR is complete?

isra-fel commented:
> @isra-fel : One of the feedback was to move our swagger spec to public repository. We weren't ready to do that last month. I've created a pull request in public repository today. If i address rest of the feedback, would we be able to merge before swagger spec PR is complete?

Nope, sorry. SDK should be generated based on merged spec.

Commit:
Add missing files

farehar commented:
@isra-fel : I've addressed all the feedback. Here's the link to our swagger spec in public repository https://github.com/Azure/azure-rest-api-specs/tree/master/specification/hybridcompute/resource-manager

isra-fel commented:
@farehar `generate.ps1` needs to be checked in too

isra-fel commented on sdk/hybridcompute/Microsoft.Azure.Management.HybridCompute/src/Microsoft.Azure.Management.HybridCompute.csproj:
```diff
@@ -0,0 +1,25 @@
+  <PropertyGroup>
+    <PackageId>Microsoft.Azure.Management.HybridCompute</PackageId>
+    <Description>Provides developers with libraries for managing hybrid machines</Description>
+    <Version>0.0.1</Version>
```
First version should be `0.1.0` please. And I believe your module will be in preview, right?
```suggestion
    <Version>0.1.0-preview</Version>
```

farehar replied:
yes. updated.

isra-fel commented on sdk/hybridcompute/Microsoft.Azure.Management.HybridCompute/Properties/AssemblyInfo.cs:
```diff
@@ -0,0 +1,18 @@
+[assembly: AssemblyTitle("Microsoft Azure Hybrid Compute Management Library")]
+[assembly: AssemblyDescription("Provides management functionality for Microsoft Azure Hybrid Compute Resources.")]
+
+[assembly: AssemblyVersion("1.0.0.0")]
```
```suggestion
[assembly: AssemblyVersion("0.1.0.0")]
```

farehar replied:
updated

isra-fel commented on sdk/hybridcompute/Microsoft.Azure.Management.HybridCompute/Properties/AssemblyInfo.cs:
```diff
@@ -0,0 +1,18 @@
+[assembly: AssemblyDescription("Provides management functionality for Microsoft Azure Hybrid Compute Resources.")]
+
+[assembly: AssemblyVersion("1.0.0.0")]
+[assembly: AssemblyFileVersion("1.0.0.0")]
```
```suggestion
[assembly: AssemblyFileVersion("0.1.0.0")]
```

farehar replied:
updated

isra-fel reviewed (commented):
@farehar Please CIL thanks

Commit:
update version numbers
